### PR TITLE
Feat: add update operator for all

### PR DESCRIPTION
**Pull request by nachomazzara**
- Add `UpdateOperatorForAll` for LAND and Estate registry smart contracts

Interface:

```javascript
setUpdateManager(address _owner, address _operator, bool _approved)

event UpdateManager(
    address indexed _owner,
    address indexed _operator,
    address indexed _caller,
    bool _approved
); 
```

### CODE REVIEW

File: contracts/estate/EstateRegistry.sol [hunk 1]
```diff
@@ -39,6 +39,15 @@ contract EstateRegistry is Migratable, IEstateRegistry, ERC721Token, ERC721Recei
     _;
   }
 
+  modifier onlyManager(uint256 estateId) {
```
Comment (eordano) [inline contracts/estate/EstateRegistry.sol]: canManageOperators?

Comment (nachomazzara) [reply contracts/estate/EstateRegistry.sol]: Hmm idk. It involves also to update content. I thought `canManage` but is super generic

Comment (eordano) [reply contracts/estate/EstateRegistry.sol]: "OnlyManager" can be interpreted as "not the owner, not the operator, only the manager".

Comment (eordano) [reply contracts/estate/EstateRegistry.sol]: canSetUpdateOperator

**Comment by nachomazzara** [comment]
@eordano ready to merge

**Comment by eordano** [comment]
circleci está de vacas

**Pull request merged**